era5cli/utils.py

```diff
@@ -183,3 +183,22 @@ def _append_netcdf_history(ncfile: str, appendtxt: str):
     except AttributeError:
         ncfile.history = appendtxt
     ncfile.close()
+
+
+def strtobool(value: str) -> bool:
+    """Convert a string to a boolean. Required to have a true/false arg in argparse.
+
+    For example: `--flag True` or `--flag False`.
+    """
+    trues = ["true", "yes", "y", "1"]
+    falses = ["false", "no", "n", "0"]
+
```

Comment:
Very similar to [this answer](https://stackoverflow.com/a/43357954). Could consider linking to it to have a bit of context. Note they also allow `t`/`f`, do you want to add that?

Reply:
It's also similar to https://github.com/symonsoft/str2bool which is older. And the original list of trues/falses is from https://docs.python.org/3/distutils/apiref.html#distutils.util.strtobool. Their implementation is [here](https://github.com/pypa/distutils/blob/4435cec31b8eb5712aa8bf993bea3f07051c24d8/distutils/util.py#L340-L353).

I guess I might as well add t/f to comply with that.